src/controllers/factorycontroller.js

```diff
@@ -48,6 +48,19 @@ class FactoryController extends OrderController
             }
         });
 
+        //increases the truck size (orderCapacity) after cost validation
+        $("#size-truck").click(() => {
+            if (GAME.model.config.money > GAME.model.config.costSizeTruck) {
+                MoneyController.updateMoney(-GAME.model.config.costSizeTruck);
+                GAME.model.config.orderCapacity*GAME.model.config.sizeIncreasingPercentage;
```

Comment:
This does not do anything, you will have to assign this (i.e. `GAME.model.config.orderCapacity = ...`